languages/python/exercises/concept/strings/.docs/after.md

```diff
@@ -0,0 +1,17 @@
+In this lesson, we learned:
```

Comment:
I'm not particularly fond of the word "lesson." Maybe for inspiration, see [this after.md document](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/strings/.docs/after.md)?

Reply:
The list below could perhaps also include information on how strings are immutable and how to define strings (quote-wise). Alternatively, most tracks don't use a bullet list for the lessons but use a bit more text (see [this after.md document](https://github.com/exercism/v3/blob/master/languages/csharp/exercises/concept/strings/.docs/after.md)).